readme.md

```diff
@@ -105,12 +124,26 @@ Then, make each route respond with a one-word string containing the RESTful acti
 ```js
 var express = require("express");
 var app = express();
+var bodyParser = require("body-parser");
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.post("/", function(req, res){
+  res.send("Posted"+req.body.name);
+});
+app.get("/", function(req, res){
+  res.send("Read");
+});
+app.update("/edit", function(req, res){
+  res.send("Edited"+req.body.name);
```

Comment:
Not quite -- remember we want to be RESTful here and we have 5 http verbs to use.